helix-term/src/handlers/signature_help.rs

```diff
@@ -231,26 +229,8 @@ pub fn show_signature_help(
     };
     contents.set_active_param_range(active_param_range());
 
-    let old_popup = compositor.find_id::<Popup<SignatureHelp>>(SignatureHelp::ID);
-    let mut popup = Popup::new(SignatureHelp::ID, contents)
-        .position(old_popup.and_then(|p| p.get_position()))
-        .position_bias(Open::Above)
-        .ignore_escape_key(true);
-
-    // Don't create a popup if it intersects the auto-complete menu.
-    let size = compositor.size();
-    if compositor
-        .find::<ui::EditorView>()
-        .unwrap()
-        .completion
-        .as_mut()
-        .map(|completion| completion.area(size, editor))
-        .filter(|area| area.intersects(popup.area(size, editor)))
-        .is_some()
-    {
-        return;
-    }
-
+    let old_popup = compositor.find_id::<SignaturePopup>(SignatureHelp::ID);
+    let popup = SignaturePopup::new(contents, old_popup.and_then(|p| p.get_position()));
```

Comment:
I do prefer the `Above` bias for signature help. Why was that moved underneath the cursor? Particularly with autcompletions now being almost always below that just seems like the natural choice.

Reply:
It is still above. the `SignaturePopup` is new struct it will set itself Above in the `new` method. but this is not important anymore, since this is reverted